docs/node-mixin/dashboards/network.libsonnet

```
@@ -0,0 +1,747 @@
+local grafana = import 'github.com/grafana/grafonnet-lib/grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+local nodePanels = import '../lib/panels/panels.libsonnet';
+local commonPanels = import '../lib/panels/common/panels.libsonnet';
+local nodeTimeseries = nodePanels.timeseries;
+local common = import '../lib/common.libsonnet';
+
+{
+
+  new(config=null, platform=null):: {
+    local c = common.new(config=config, platform=platform),
+    local commonPromTarget = c.commonPromTarget,
+    local templates = c.templates,
+    local q = c.queries,
+
+    local networkTrafficPanel =
+      commonPanels.networkTrafficGraph.new('Network Traffic')
+      .addTarget(commonPromTarget(
+        expr=q.networkReceiveBitsPerSec,
+        legendFormat='{{device}} received',
+      ))
+      .addTarget(commonPromTarget(
+        expr=q.networkTransmitBitsPerSec,
+        legendFormat='{{device}} transmitted',
+      )),
+
+    local networkPacketsPanel =
+      nodeTimeseries.new('Unicast Packets')
+      .addTarget(commonPromTarget(
+        'irate(node_network_receive_packets_total{%(nodeQuerySelector)s,}[$__rate_interval])' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='{{device}} received',
+      ))
+      .addTarget(commonPromTarget(
+        'irate(node_network_transmit_packets_total{%(nodeQuerySelector)s,}[$__rate_interval])' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='{{device}} transmitted',
+      ))
+      .withDecimals(1)
+      .withUnits('pps')
+      .withNegativeYByRegex('transmit')
+      .withAxisLabel('out(-) / in(+)'),
```

Would this make more sense with a "pipe" character, since it get's rendered vertically, and would appear more like a horizontal rule?

```suggestion
      .withAxisLabel('out(-) | in(+)'),
```